plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/TeleportCommand.java

```diff
@@ -148,6 +158,47 @@ else if (entityList.identify().startsWith("l@")) { // Compensate for legacy enti
                 NMSHandler.entityHelper.look(entity.getBukkitEntity(), location.getYaw(), location.getPitch());
                 return;
             }
+            if (offthreadRepeats != null && relativeAxes != null && entity.isPlayer()) {
+                NetworkInterceptHelper.enable();
+                int times = offthreadRepeats.asInt() + 1;
+                int ms = 50 / times;
+                Player player = entity.getPlayer();
+                Vector increment = location.clone().subtract(player.getLocation()).toVector().multiply(1.0 / times);
+                double x = relativeAxes.contains(Relative.X) ? increment.getX() : location.getX();
+                double y = relativeAxes.contains(Relative.Y) ? increment.getY() : location.getY();
+                double z = relativeAxes.contains(Relative.Z) ? increment.getZ() : location.getZ();
+                float yaw;
+                if (relativeAxes.contains(Relative.YAW)) {
+                    float relYaw = (location.getYaw() - player.getLocation().getYaw()) % 360;
+                    if (relYaw > 180) {
+                        relYaw -= 360;
+                    }
+                    yaw = offthreadYaw ? relYaw / times : 0;
+                }
+                else {
+                    yaw = location.getYaw();
+                }
+                float pitch;
+                if (relativeAxes.contains(Relative.PITCH)) {
+                    pitch = offthreadPitch ? (location.getPitch() - player.getLocation().getPitch()) / times : 0;
+                }
+                else {
+                    pitch = location.getPitch();
+                }
+                List<Relative> finalRelativeAxes = relativeAxes;
+                DenizenCore.runAsync(() -> {
+                    try {
+                        for (int i = 0; i < times; i++) {

```

Comment:
This will loop 1 too many times, per line 163 `int times = offthreadRepeats.asInt() + 1;`

Reply:
patched